### PR TITLE
Update to load-no-partial-leading-underscore

### PR DESCRIPTION
**Pull request by Alkarex**
fix https://github.com/stylelint-scss/stylelint-config-recommended-scss/issues/251
See https://github.com/stylelint-scss/stylelint-scss/blob/v5.3.0/src/rules/at-import-no-partial-leading-underscore/README.md

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ module.exports = {
 		],
 		'scss/at-extend-no-missing-placeholder': true,
 		'scss/at-if-no-null': true,
-		'scss/at-import-no-partial-leading-underscore': true,
+		'scss/load-no-partial-leading-underscore': true,
```
[the remark this answers is not in the document]
Comment (Alkarex) [reply index.js:23]: I did not want to have too many things in the same PR, so this PR contains the minimum `npm update`, which updated `package-lock.json` to that version

[the remark this answers is not in the document]
Comment (Alkarex) [reply index.js:23]: I can happily update `package.json` as well to be more clear.

[the remark this answers is not in the document]
Comment (Alkarex) [reply index.js:23]: https://github.com/stylelint-scss/stylelint-config-recommended-scss/pull/252/commits/b2b32bb758423bc99eb6739099b9e511d9794ef2

**Comment by Alkarex** [comment]
@tagliala If you want / can a bit more the consequences of the change, you can use the following in your `package.json`. Feedback welcome.

```json
"stylelint-config-recommended-scss": "git+https://github.com/Alkarex/stylelint-config-recommended-scss.git#load-no-partial-leading-underscore",
```

File: package.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
   "dependencies": {
     "postcss-scss": "^4.0.9",
     "stylelint-config-recommended": "^13.0.0",
-    "stylelint-scss": "^5.1.0"
+    "stylelint-scss": "^5.3.0"
```
Comment (tagliala) [inline package.json:31]: I think this is not correct, apologies.

I'm having this deprecation message also on 5.2.1 now. I will edit the main issue

```
Deprecation warnings:
 - The "scss/at-import-no-partial-leading-underscore" rule is deprecated.
 - 'at-import-no-partial-leading-underscore' has been deprecated, and will be removed in '6.0'. Use 'load-no-partial-leading-underscore' instead. See: https://github.com/stylelint-scss/stylelint-scss/blob/v5.2.1/src/rules/at-import-no-partial-leading-underscore/README.md
```

File: package-lock.json
Comment (tagliala) [inline package-lock.json:1]: 11,633 changes: 6,899 additions & 4,734 deletions

Looks like there are unwanted changes in this file

Comment (Alkarex) [reply package-lock.json:1]: Why? This looks correct. This is the result of a needed `npm update`

**Comment by XhmikosR** [comment]
Please make a clean PR with the needed change only.

**Comment by Alkarex** [comment]
A PR can be edited further until reaching the desired state. It is a better process than closing it.
And what do you mean more precisely with the undesired changes? You do not want any update to `package-lock.json`? Running `npm update` after changing libraries is quite normal. I can of course run a limited `npm update stylelint-scss` if you prefer, but in every case, it will end up in the exact same result after running `npm update` again...

**Comment by XhmikosR** [comment]
Like I said, keep the changes to a minimum. So, no touching unrelated files.

**Comment by Alkarex** [comment]
`package-lock.json` *is* related, since a library has been updated... Anyway, I can perform an `npm update stylelint-scss` if you prefer.
By the way, if you apply an `npm update` on your `master` branch, this PR will automagically have the correct diff.

**Comment by Alkarex** [comment]
For the record, Dependabot also updates `package-lock.json` after updating a library, just like I did: https://github.com/stylelint-scss/stylelint-config-recommended-scss/pull/257

File: package.json (hunk 2 above)
Comment (tagliala) [reply package.json:31]: @XhmikosR is this change correct? I'm seeing this warning also on 5.2.1, but from `stylelint-scss` changelog I can see that `load-no-partial-leading-underscore` has been introduced in 5.3.0 via stylelint-scss/stylelint-scss#867

**Comment by XhmikosR** [comment]
There's a difference in one running `npm update` in a PR and updating **only** the needed dependency. Dependabot splits PRs and we can see the diff clearly.

The right PR here would be #259 plus #254. Everything else shouldn't be here.

**Comment by Alkarex** [comment]
So two PRs instead of one?
That does not explain why you closed my PR. See https://docs.github.com/articles/about-pull-request-reviews/
Anyway, moving on.

**Pull request closed without merging**